convert_hf_to_gguf.py

```diff
@@ -4239,6 +4239,44 @@ def set_vocab(self):
         super().set_vocab()
 
 
+@ModelBase.register("Qwen3OmniMoeForConditionalGeneration")
+class Qwen3OmniThinkerModel(Qwen3MoeModel):
```

Comment:
```suggestion
class Qwen3OmniThinkerModel(Qwen2MoeModel):
```
Let's not nest these too deeply, `Qwen3MoeModel` just contains `InternS1` special-casing.